tools/cawlign/examples/v0.1.0.html

```diff
@@ -0,0 +1,18 @@
+<script src="https://biowasm.com/cdn/v3/aioli.js"></script>
+<script type="module">
+const CLI = await new Aioli(["cawlign/0.1.0"]);
+
+// Align a sequence in test.fas against a reference genome
+const output = await CLI.exec("cawlign -o test.aln -r /shared/cawlign/res/references/HXB2_pol /shared/cawlign/HIV1-pol-326.fa");
```

Comment:
Interestingly, looks like `--preload-file res@/cawlign` mounts the 2 folders inside `res` directly at `/cawlign`, but doesn't include the `res` folder, so I had to change it to the following to get it working:

```suggestion
const output = await CLI.exec("cawlign -o test.aln -r /shared/cawlign/references/HXB2_pol /shared/cawlign/HIV1-pol-326.fa");
```